app/pages/components/LibraryPage.js

```diff
@@ -7,22 +7,31 @@ import { connect } from 'react-redux';
 
 import type { State } from 'types/state';
 import type { Identifier } from 'types/model';
-import topics from 'modules/topics';
-import { CURRENT_USER } from 'modules/users/constants';
 import { Button, Grid } from 'semantic-ui-react';
 import { Link } from 'react-router-dom';
 
+import topics from 'modules/topics';
+import authentication from 'modules/authentication';
+
 import Page from '../Page';
 
+const { getAllTopicIdsByUserId } = topics.selectors;
+const { getAccount } = authentication.selectors;
+
 type StateProps = {
   topicIds: Array<Identifier>,
 };
 
 type Props = CustomTranslatorProps & StateProps;
 
 const mapStateToProps = (state: State): StateProps => {
+  const account = getAccount(state);
+
+  // TODO: does this need null checks or is it impossible to access when not logged in?
+  const CURRENT_USER = account != null ? account.id : 'jantje1234';
```

Comment:
When the authentication module is integrated, every page that requires authentication will check for it; if the client is not authenticated then the browser is redirected to sign in. Null checks are not necessary.